js/components/record/TypeaheadAnswer.js

```diff
@@ -58,7 +58,7 @@ const TypeaheadAnswer = (props) => {
             isSearchable={true}
             isLoading={isLoading}
             isClearable={true}
-            isDisabled={isLoading}
+            isDisabled={isLoading || (props.isDisabled === undefined ? false : props.isDisabled) }
```

Comment:
```suggestion
            isDisabled={isLoading || (props.isDisabled === true) }
```

Reply:
Moreover I believe it should be mentioned in `TypeaheadAnswer.propTypes`. Right @LaChope ?

Reply:
@blcham 
Resolved in commit [514bc80](https://github.com/kbss-cvut/record-manager-ui/pull/33/commits/514bc807bc185ba34f80b73b1bcb08d2e244092f)